include/toolkitAPI.h

```diff
@@ -488,6 +489,12 @@ int DLLEXPORT swmm_getSimulationDateTime(int timetype, int *year, int *month,
 */
 int DLLEXPORT swmm_setSimulationDateTime(int timetype, char *dtimestr);
 
+/**
+ @brief Save hotstart file during simulation
+ @param hsfile The file name of the hotstart file that user would like to save 
+*/
+void DLLEXPORT save_hotstart(char *hsfile);
```

Comment:
Sticking with the convention, please pre-pend the exportable functions with `swmm_*`

Reply:
Yes. I will do that.